### DISCUSSION TITLE
Multiple Rates By Provider Role

### DISCUSSION THREAD

**Discussion by TamHaynes** [CMSgov/price-transparency-guide#730] [General]
We have multiple negotiated rates for a provider based on the role the provider is performing. How would we indicate the role on the in-network-rate.json file? Please advise.

**Comment by theogaraj** [comment]
@TamHaynes can you elaborate on what you mean by "based on the role the provider is performing"?  An example would help to connect this to a real-world scenario!

**Reply by TamHaynes** [reply]
Here is an example of the values 
Anesthesiology
Diagnostic Radiology
Exception
Mental Health
Mental Health 1/2
Mental Health 3/4
Nurse Anesthetist, Certified Registered 
Pathology
Standard Reimbursement
So we would have a provider that could get either "Standard Reimbursement" or "Mental Health" 
They would  be listed with both pricing agreements but would be priced based on the "Mental Health" or "Standard Reimbursement"
Therefore I would like to show that service code "05" would have $22.57 for "Standard Reimbursement" but would have $28.75 for "Mental Health"
Otherwise I am just sending the service code "05" with two different prices and you don't know which one to use. 
Does that make sense?

**Reply by tdfow** [reply]
The Additional Information field can be used to clarify any pricing for the service being published, when needed.

**Reply by nmayle** [reply]
@TamHaynes in this case, does the rate not also depend on some other code e.g. CPT, HCPCS...?
